
From: Sean Neakums <sneakums@zork.net>

This patch kills two printks from UDF that announce its registration and
unregistration.  Since one can determine which filesystems are present by
examining /proc/filesystems, these messages strike me as noise.

Signed-off-by: Sean Neakums <sneakums@zork.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/udf/super.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN fs/udf/super.c~kill-udf-registration-unregistration-messages fs/udf/super.c
--- 25/fs/udf/super.c~kill-udf-registration-unregistration-messages	Mon Aug  2 15:53:52 2004
+++ 25-akpm/fs/udf/super.c	Mon Aug  2 15:53:52 2004
@@ -194,7 +194,6 @@ struct udf_options
 static int __init init_udf_fs(void)
 {
 	int err;
-	printk(KERN_NOTICE "udf: registering filesystem\n");
 	err = init_inodecache();
 	if (err)
 		goto out1;
@@ -210,7 +209,6 @@ out1:
 
 static void __exit exit_udf_fs(void)
 {
-	printk(KERN_NOTICE "udf: unregistering filesystem\n");
 	unregister_filesystem(&udf_fstype);
 	destroy_inodecache();
 }
_
